src/main/java/lesson/lesson2/Task1/Entity/HeavyPlane.java

```diff
@@ -1,4 +1,4 @@
-package lesson.lesson2.Task1;
+package lesson.lesson2.Task1.Entity;
```

Comment:
A package name should start with with a lowercase letter, it helps avoid misunderstanding(package or class)